app/scaffold/scaffold.js

```diff
@@ -189,7 +194,7 @@ class Scaffold {
             const orderCipherText = DOM.orderCipherTextElement.value;
 
             if (!orderId || !orderCipherText) {
-                return DOM.alertBox("Please enter an order id and an order to decrypt");
+                return alertBox.show("Please enter an order id and an order to decrypt");
```

Comment:
Should these messages be broken out into the message object for reusability/ predictability?

Reply:
Okay I broke out most of the messages and left some because they are part of the functions the user is going to copy and paste.

Reply:
Cool - want to push up your changes so that I can re-review?